examples/plotting/plot_rectangle.py

```diff
@@ -0,0 +1,100 @@
+"""
+==========================================================
+Drawing a Rectangle on a map
+==========================================================
+
+In this example we are going to look at how we can plot rectangles on maps
+while describing the dimensions using different methods.
+"""
+
+################################################################################
+# Fundamentally, we will be using the `~sunpy.map.GenericMap.draw_rectangle` method to
+# draw the rectangles. Its function signature is like so:
+#
+# `draw_rectangle(bottom_left, *, width: Unit(‘deg’) = None,
+# height: Unit(‘deg’) = None, axes=None, top_right=None, **kwargs)`
+#
+# All of the extra kwargs are passed on to the `~matplotlib.patches.Rectangle` instance,
+# which allows us to modify the color, linestyle, etc. of the plotted rectangle.
+# A full overview of available options can be found in the Rectangle documentation.
```

Comment:
I think this is redundant. You've already said this above and the intersphinx mapping links to the matplotlib API docs for `Rectangle`.